src/actions/range.js

```diff
@@ -57,7 +58,7 @@ export const deleteRange = id => ({
   },
 });
 
-export const loadRanges = state => ({
+export const loadRanges = ranges => ({
   type: LOAD_RANGES,
-  state,
+  state: ranges,
```

Comment:
should the key not change too in this case?

Reply:
https://github.com/digirati-co-uk/timeliner/pull/105/commits/db60abc54cd7a5a0359d79e66d39a418d068f28b